src/test/java/fi/helsinki/cs/tmc/core/commands/SendFeedbackTest.java

```diff
@@ -22,17 +22,19 @@
 
 public class SendFeedbackTest {
 
-    private static final String URL = "http://localhost:8080/feedback";
+    private static String URL = "http://localhost:8080/feedback";
```

Comment:
No reason to have this as static. Just generates inter-test state that future changes can break. 

Is it possible to simply replace all usages of `URL` in this file with `serverAddress`?